prow/config.yaml

```diff
@@ -12719,115 +12468,309 @@ periodics:
+
+- interval: 24h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-aws-weave
+  labels:
+    preset-service-account: true
+    preset-aws-ssh: true
+    preset-aws-credential: true
+  spec:
+    containers:
+    - args:
+      - --timeout=140
+      - --bare
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180306-47f8a1178-master
+
+- interval: 30m
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-gce
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=140
+      - --bare
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180306-47f8a1178-master
+
+- interval: 1h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-gce-channelalpha
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=140
+      - --bare
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180306-47f8a1178-master
+
+- interval: 30m
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-gce-ha
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=140
+      - --bare
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180306-47f8a1178-master
+
+- interval: 2h
+  name: ci-kubernetes-e2e-kubeadm-gce
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180306-1db99ab1d
+      args:
+      - "--repo=k8s.io/kubernetes=master"
+      - "--clean"
+      - "--timeout=320"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+- name: ci-kubernetes-e2e-kubeadm-gce-1-10
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180306-1db99ab1d
+      args:
+      - "--repo=k8s.io/kubernetes=release-1.10"
+      - "--clean"
+      - "--timeout=320"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+
+- name: ci-kubernetes-e2e-kubeadm-gce-1-7
+  interval: 12h
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180306-1db99ab1d
+      args:
+      - "--repo=k8s.io/kubernetes=release-1.7"
+      - "--clean"
+      - "--timeout=320"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+- name: ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8
+  interval: 12h
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180306-1db99ab1d
+      args:
+      - "--repo=k8s.io/kubernetes=release-1.8"
+      - "--timeout=320"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+# TODO(bentheelder): switch ci-kubernetes-bazel-build to 100% periodic,
+# like the other build job once we don't have run_after_success
```

Comment:
Outdated comment?

Reply:
this still needs to be considered, it just got automatically moved by config-sort :/